### DISCUSSION TITLE
imapsync missing scripts

### DISCUSSION THREAD

**Discussion by xavier7179** [Homebrew/discussions#1538] [Everyday usage]
Hello,

I do not know how to make it a bug without issue but the imapsync tool installation is missing a link to the W/tools folder, i.e., https://github.com/imapsync/imapsync/blob/master/W/tools/fix_email_for_exchange.py
Looking at the discussion on the project GitHub page, those tools are useful to solve some synchronization issues. How should this kind of issue being notified to the Homebrew maintainers?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew has all the information you need to resolve this

**Reply by xavier7179** [reply]
Thank you  but the point was notifying someone about that to let her/him handle the pull request. Unfortunately, I’m not good at those stuff

**Reply by SMillerDev** [reply]
Homebrew does not have dedicated maintainers for specific software. We depend on the public to send pull request when they notice something
